softlayer/resource_softlayer_virtual_guest.go

```diff
@@ -615,6 +620,12 @@ func resourceSoftLayerVirtualGuestCreate(d *schema.ResourceData, meta interface{
 		return err
 	}
 
+	// Set notes
+	err = setNotes(id, d, meta)
+	if err != nil {
+		return err
+	}
+
```

Comment:
I tried setting it in `getVirtualGuestTemplateFromResourceData()` in the struct 
```
opts := datatypes.Virtual_Guest{
...
Notes: sl.String(d.Get("notes").(string))
}
```

But `notes` did not populate upon provision.